seshdash/generateData.py

```diff
@@ -0,0 +1,50 @@
+from django.utils import timezone
+from seshdash.data.db.influx import Influx
+from datetime import timedelta, datetime
+from django.forms import model_to_dict
+from seshdash.tasks import run_aggregate_on_historical
+from seshdash.models import Daily_Data_Point, BoM_Data_Point, Sesh_Site
+from seshdash.utils.time_utils import get_time_interval_array
+from seshdash.tests.data_generation import get_random_int, get_random_binary, get_random_interval, generate_date_array, get_random_float
+
+
+def create_test_data(site_id):
```

Comment:
Please be a bit more specific with the function name. This function is generating test data for what? Individual days or data points? The name should be descriptive to this